### DISCUSSION TITLE
How to edit Modbus ID of a device in script?

### DISCUSSION THREAD

**Discussion by Github-User-002** [frangoteam/FUXA#1494] [Q&A]
Hi
I have a need to edit the Modbus ID and the tcp adress of mqtt device  from a script. How to do?
I have a system that has 3 modes, I want to use the same front end for this but I have to edit the devices manually in the fuxa editor/devices page now. It is a problem for me as the things are not at my place but have another user.
THe modbus ID must be a parameter as I have a crash in the system from other devices on the network. and it is easier to just have the remote person tun off the conflicting device and write new settings for the fuxa devices from an input field in a settings page on the live screen. there is no keyboard on the system and the blue dot for editor.. is removed as the user is not supposed to edit anything major in the system.......

**Comment by unocelli** [comment]
Hi, this feature is missing, please add it in the issues section
